packages/frontend-2/components/header/NavBar.vue

```diff
@@ -85,25 +86,18 @@
       </div>
     </div>
 
-    <DisclosurePanel class="sm:hidden">
-      <div class="space-y-1 pt-2 pb-4">
-        <DisclosureButton
-          v-for="item in navigation"
-          :key="item.name"
-          as="a"
-          :href="item.href"
-          :class="[
-            item.current
-              ? 'bg-background-2 border-primary-lighter text-foreground-2'
-              : 'border-transparent text-foreground-3 hover:text-foreground-2 hover:bg-background-2 hover:border-foreground-4',
-            'block pl-4 pr-4 py-2 border-l-4 text-base font-medium'
-          ]"
-          :aria-current="item.current ? 'page' : undefined"
-        >
-          {{ item.name }}
-        </DisclosureButton>
+    <DisclosurePanel class="sm:hidden" on-pointerleave="">
+      <div class="flex flex-col space-y-6 pt-2 pb-6 px-5">
+        <HeaderNavLink to="/" name="Dashboard" class="" />
+        <HeaderNavLink
+          v-for="(nl, i) in nav.filter((n) => !!n)"
```

Comment:
Duplicate template should be cleaned up at some point (we can leave this in while we're in the prototyping phase)